src/test/java/org/terracotta/statistics/simulation/latency/LongSample.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * All content copyright Terracotta, Inc., unless otherwise indicated.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics.simulation.latency;
+
+import java.io.Serializable;
+
+/**
+ * @author Mathieu Carbou
+ */
+public class LongSample implements Serializable, Comparable<LongSample> {
```

Comment:
Can we not just use a Sample<Long>?

Reply:
I would have liked but i do not like sample for 2 reasons:

1. It works on Long and not long
2. I do not like the name `getTimestamp()` : what I am using is a relative time, not a timestamp
3. I do not like the name `getSample()` : because we are in a `Sample` class so I would much more expect 2 methods: time and value.

I did not change the `Sample` class because it would require a broader change and I don't know if people are OK with that.

ideally I would like to have a Sample<T> class with a time and value.

Reply:
I'll add another separate commit to try use this existing class though. But I do not like it.